tor_archivist/core/blossom.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+class ConfigurationError(Exception):
+    pass
+
+class BlossomAPI(object):
+    def __init__(
+            self,
+            email: str=None,
+            password: str=None,
+            api_key: str=None,
+            api_base_url: str="http://api.grafeas.localhost:8000",
+            login_url: str="http://grafeas.localhost:8000/login/"
+    ):
+        if not email:
+            raise ConfigurationError("Need an email address!")
+        if not password:
+            raise ConfigurationError("Need a password!")
+        if not api_key:
+            raise ConfigurationError("Need an API key!")
+
+        self.email = email
+        self.password = password
+        self.base_url = api_base_url
+        self.login_url = login_url
```

Comment:
The API base url and login url seem like they don't need to be passed in if you use `os.getenv()`. I like defaulting to a local dev setup so the fallback values (noted in the args) are fine as-is, but I'd like to simplify how this object is instantiated.

Reply:
@itsthejoker thoughts?

Reply:
Yes, this should definitely change and this is a good fix. For now though it's probably fine.